mongo/document/base.go

```diff
@@ -33,10 +33,6 @@ func (d *Base) ensureCreatedAt() {
 	}
 }
 
-func (d *Base) setCreatedAt(t time.Time) {
```

Comment:
Woot? Why did you remove that?

Reply:
Not used

Reply:
`setUpdatedAt` seems also useless, don't you think?

Reply:
Nop it's mendatory to implement the `document` interface.